drivers/sensor/renesas/hs400x/hs400x.c

```diff
@@ -137,15 +137,45 @@ static int hs400x_channel_get(const struct device *dev, enum sensor_channel chan
 	return 0;
 }
 
+static void hs400x_all_measurements_stop(const struct device *dev)
```

Comment:
Shouldn't this return the results from the i2c operations?

Reply:
The commands may return ACK (If a periodic measurement is running or a no hold measurement is already started) or NACK (If a periodic measurement is not running or a no hold measurement isn't issued).
In both cases, no measurement is running.